pages/common/ffplay.md

```diff
@@ -7,6 +7,14 @@
 
 `ffplay {{path/to/file}}`
 
+- Play audio from a media file without a GUI:
+
+`ffplay -nodisp {{path/to/file}}`
+
+- Play media passed by ffmpeg through standard stream:
```

Comment:
```suggestion
- Play media passed by `ffmpeg` through standard stream:
```

Reply:
That doesn't convey well enough that it supports protocols in my opinion. Like rtsp://